scout/parse/variant/callers.py

```diff
@@ -28,36 +36,51 @@ def parse_callers(variant, category="snv"):
     svdb_origin = variant.INFO.get("svdb_origin")
     raw_info = variant.INFO.get("set")
 
+    filter_status_default = "Pass"
+    if variant.FILTER is not None:
+        filter_status_default = "Filtered - {}".format(variant.FILTER.replace(";", " - "))
+
     if other_info:
-        for info in other_info.split(","):
+        infos = other_info.split(",")
+        if len(infos) > 1:
+            filter_status_default = "Pass"
+        for info in infos:
             called_by = info.split("|")[0]
             if called_by in callers_keys:
-                callers[called_by] = "Pass"
+                callers[called_by] = filter_status_default
     elif svdb_origin:
-        for called_by in svdb_origin.split("|"):
+        callers = svdb_origin.split("|")
+        if len(callers) > 1:
+            filter_status_default = "Pass"
+        for called_by in callers:
             if called_by in callers_keys:
-                callers[called_by] = "Pass"
+                callers[called_by] = filter_status_default
     elif raw_info:
         info = raw_info.split("-")
         for call in info:
             if call == "FilteredInAll":
+                filter_status = filter_status_default
+                if "Filtered" not in filter_status_default:
+                    filter_status = "Filtered"
                 for caller in callers:
-                    callers[caller] = "Filtered"
+                    callers[caller] = filter_status
             elif call == "Intersection":
                 for caller in callers:
-                    callers[caller] = "Pass"
+                    callers[caller] = filter_status_default
             elif "filterIn" in call:
+                filter_status = filter_status_default
+                if "Filtered" not in filter_status_default:
+                    filter_status = "Filtered"
                 for caller in callers:
                     if caller in call:
-                        callers[caller] = "Filtered"
+                        callers[caller] = filter_status
             elif call in callers_keys:
-                callers[call] = "Pass"
+                callers[call] = filter_status_default
 
     if raw_info or svdb_origin or other_info:
         return callers
 
     if category == "snv":
-        # cyvcf2 FILTER is None if VCF file column FILTER is "PASS"
         filter_status = "Pass"
         if variant.FILTER is not None:
             filter_status = "Filtered - {}".format(filter_status.replace(";", " - "))
```

Comment:
Note that `filter_status` could never take any other value than "Pass" here really, so it was either "Pass" or "Filtered - Pass" I guess?!